test/specs/cli.js

```diff
@@ -134,6 +134,15 @@ describe('cli', () => {
       output.code.should.equal(2);
     });
 
+    it('uses --negate to allows for "not" conditions ', () => {
+      const woNegate = cli('test', '-d', 'fakeDirName');
+      woNegate.code.should.equal(1);
+      const wNegate = cli('--negate', 'test', '-d', 'fakeDirName');
+      wNegate.stdout.should.equal('');
+      wNegate.stderr.should.equal('');
+      wNegate.code.should.equal(0);
```

Comment:
Can you add another test case where the command normally succeeds but `--negate` gives it a status of `1`? The `true` command should be a good example to use for that case.

Reply:
Done !